site/fetchers/featured-blog-posts/index.js

```diff
@@ -0,0 +1,61 @@
+/* @flow */
+
+import fetch from 'node-fetch';
+import type { BlogPost } from '../../pages/home/blog-slice/blog-entry';
+
+const getQueryString = params => {
+  return Object.keys(params).map(
+    k => `${encodeURIComponent(k)}=${encodeURIComponent(params[k])}`
+  ).join('&');
+};
+
+const getUrl = params => {
+  return `https://blog.red-badger.com/blog/?&format=json&${getQueryString(params)}`;
+};
+
+const handleErrors = response => {
+  if (!response.ok) {
+    throw Error(response.statusText);
+  }
+
+  return response;
+};
+
+export const sanitiseAuthorBio = (bio: string = ''): string => { // eslint-disable-line arrow-parens,max-len
+  const role = bio.match(/<.+>(.*)<.+>/);
+  return (role && role[1]) || bio;
+};
+
+export const mapDataToState = (data: Object): Array<BlogPost> => ( // eslint-disable-line arrow-parens,max-len
+  data.map((post: Object): BlogPost => ({ // eslint-disable-line arrow-parens
```

Comment:
If this is going to happen every time with flow annotations we'd better just turn this rule off